src/zenml/cli/user_management.py

```diff
@@ -77,34 +81,51 @@ def describe_user(user_name_or_id: Optional[str] = None) -> None:
 
 
 @user.command("list")
-@list_options(UserFilter)
+@list_options(UserFilter, default_columns=["active", "id", "name"])
```

Comment:
The name refers to the email here. I would suggest adding the actual name and the role as well.